src/test/unit/requests/hearingRequests.test.ts

```diff
@@ -33,3 +42,62 @@ describe('Hearing get requests', () => {
     expect(await hearingRequests.getHearingsByCaseName('bob')).toStrictEqual([]);
   });
 });
+
+describe(`getHearingByCaseReferenceNumber(${validCaseNo})`, () => {
+  beforeEach(() => {
+    stub.withArgs('/hearings/case-number/').resolves(Promise.reject(errorResponse));
+    stub.withArgs('/hearings/case-number/fail').resolves(Promise.reject(errorRequest));
+    stub.withArgs('/hearings/case-number/ABC12345').resolves({data: subscriptionsCaseData});
+  });
+
+  it('should return list of cases', async () => {
+    return hearingRequests.getHearingByCaseReferenceNumber(validCaseNo).then(data => {
+      expect(data).toBe(subscriptionsCaseData);
+    });
+  });
+
+  it('should return list of 1 case', () => {
```

Comment:
this doesn't bring any value as you are already comparing data in **should return list of cases**

Reply:
remove duplicate test.